plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -1196,8 +1215,7 @@ mark.amount {
 	}
 }
 
-h2 .woocommerce-help-tip,
-.woocommerce-product-type-tip {
+h2 .woocommerce-help-tip {
```

Comment:
`.woocommerce-product-type-tip` seems to be used only in `#woocommerce-product-data`